scripts/karma.coffee

```diff
@@ -26,18 +26,17 @@ class Karma
       users: []
       , things: []
     }
-    @allowances = {}
-    @karma_allowance = 10
+    @khistory = []
 
     @increment_responses = [
-      "+1!", "killin' it!", "is en fuego!", "leveled up!"
+      "FTW!", "killin' it!", "is en fuego!", "leveled up!", "ROCK-N-ROLL!"
     ]
 
     @robot.brain.on 'loaded', =>
       if @robot.brain.data.karma
         @cache = @robot.brain.data.karma
-      if @robot.brain.data.karmaAllowances
-        @allowances = @robot.brain.data.karmaAllowances
+      if @robot.brain.data.karmaHistory
+        @khistory = @robot.brain.data.karmaHistory
       @karma_allowance ?= process.env.KARMA_ALLOWANCE
```

Comment:
This should be changed to be the allowance of total karmas in the @khistory array for that user. A user should be capped around ~1000 for a seven day period. Maybe both the count and the period day values should be variables.